### PR TITLE
Added a bounce-and-blow animation

### PR DESCRIPTION
**Pull request by Graviton5**

### CODE REVIEW

**Comment by NiallEccles** [comment]
Thanks for opening this pull request. Couple of issues with your code that I will highlight now.

**Comment by Graviton5** [comment]
Yes please, let me know.

**Review by NiallEccles — CHANGES_REQUESTED** [review]

File: animations/bounce-open.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	animation-iteration-count: 4;
+}
+
+.scaler {
+	position: absolute;
+	animation: expand 2s forwards;
+	animation-timing-function: ease-in;
+	animation-delay: 3s;
+	animation-iteration-count: 1;
+}
```
[the remark this answers is not in the document]
Comment (Graviton5) [reply animations/bounce-open.css:32]: I will change the class name if you require it. The scaler class is used to do the blow animation to the ball (bounce-open class)

**Comment by Graviton5** [comment]
Updated the file with the reuqested changes

**Pull request merged**